<?xml version="1.0" ?>
<robot name="box">
    <material name="DHL_color">
        <color rgba="1 0.8 0 1"/>
    </material>

    <link name="box_corpus">
        <collision>
            <origin xyz="0 0 -0.1675" rpy="0 0 0"/>
            <geometry>
                <box size="0.34 0.32 0.005"/>
            </geometry>
            <material name="DHL_color"/>
        </collision>
        <collision>
            <origin xyz="0.167578  0 -0.0025" rpy="0 0 0"/>
            <geometry>
                <box size="0.005156 0.32 0.335"/>
            </geometry>
            <material name="DHL_color"/>
        </collision>
        <collision>
            <origin xyz="-0.167578  0 -0.0025" rpy="0 0 0"/>
            <geometry>
                <box size="0.005156 0.32 0.335"/>
            </geometry>
            <material name="DHL_color"/>
        </collision>
        <collision>
            <origin xyz="0 0.1575 -0.0025" rpy="0 0 0"/>
            <geometry>
                <box size="0.340312 0.005 0.335"/>
            </geometry>
            <material name="DHL_color"/>
        </collision>
        <collision>
            <origin xyz="0 -0.1575 -0.0025" rpy="0 0 0"/>
            <geometry>
                <box size="0.340312 0.005 0.335"/>
            </geometry>
            <material name="DHL_color"/>
        </collision>
    </link>
    <joint name="box_flap_1_joint" type="revolute">
        <origin xyz="0.17 0 0.166" rpy="0 0 0"/>
        <parent link="box_corpus"/>
        <child link="box_flap_1"/>
        <axis xyz="0 1 0"/>
        <limit effort="1000.0" lower="-1.57" upper="4.7" velocity="3.1415"/>
    </joint>
    <link name="box_flap_1">
        <collision>
            <geometry>
                <mesh filename="package://mujoco_sim/model/ontology/box/stl/BoxFlap1.stl"/>
            </geometry>
            <material name="DHL_color"/>
        </collision>
    </link>
    <joint name="box_flap_2_joint" type="revolute">
        <origin xyz="-0.17 0 0.168" rpy="0 0 0"/>
        <parent link="box_corpus"/>
        <child link="box_flap_2"/>
        <axis xyz="0 -1 0"/>
        <limit effort="1000.0" lower="-1.57" upper="4.7" velocity="3.1415"/>
    </joint>
    <link name="box_flap_2">
        <collision>
            <geometry>
                <mesh filename="package://mujoco_sim/model/ontology/box/stl/BoxFlap2.stl"/>
            </geometry>
            <material name="DHL_color"/>
        </collision>
    </link>
</robot>
